src/ert/_c_wrappers/enkf/config/enkf_config_node.py

```diff
@@ -396,36 +326,36 @@ def create_field(
             value_max = max_key
             truncation = truncation | EnkfTruncationType.TRUNCATE_MAX
 
-        config_node = cls._alloc_field_node(key, grid, forward_init)
+        config_node = cls._alloc_field_node(key, grid)
         if config_node is None:
```

Comment:
Another error condition not hit by tests that we may want to be `ConfigValidationError`.

Reply:
Same as before.